doc/plans/plans.md

```diff
@@ -7,8 +7,7 @@ All of these plans can be used directly by the RunEngine (`RE()`) or within a wr
 So you can do this: 
 
 ```python
-
-    >>> result = RE(refl_scan("S1VG", 1, 10, 21, model=Gaussian().fit() frames=500, det=100, mon=3, pixel_range=6, periods=True, save_run=False))
+result = RE(refl_scan("S1VG", 1, 10, 21, model=Gaussian().fit() frames=500, det=100, mon=3, pixel_range=6, periods=True, save_run=False))
```

Comment:
i guess it would make more sense to call motor_scan here now we've moved refl_scan out to another docs page - minor thing really